### PR TITLE
Fix single line docsting

### PR DESCRIPTION
**Pull request by ksdme**

### CODE REVIEW

**Comment by ksdme** [comment]
Since the test environment uses the same coala config as that of the repo, the tests have to be updated for the build to pass.

**Comment by ksdme** [comment]
gitmate seems to be using a previous version of coafile from this PR.

**Comment by jayvdb** [comment]
See comments on issue.

**Comment by ksdme** [comment]
@jayvdb please review.

File: coala_langserver/jsonrpc.py [hunk 1]
```diff
@@ -78,8 +78,10 @@ def _receive(self):
         self._msg_buffer[obj.get('id') or uuid.uuid4()] = obj
 
     def read_message(self, _id=None):
-        """Read a JSON RPC message sent over the current connection. If
-        id is None, the next available message is returned."""
+        """
+        Read a JSON RPC message sent over the current connection. If
```
Comment (jayvdb) [inline coala_langserver/jsonrpc.py]: this modified version is no better than the original.

revert, ignore extra rules if required

File: .coafile [hunk 2]
```diff
@@ -14,7 +14,9 @@ language = python
 preferred_quotation = '
 
 [autopep8]
-bears = PEP8Bear, PycodestyleBear
+bears = PEP8Bear, PycodestyleBear, PyDocStyleBear
+pydocstyle_ignore = D100, D101, D102, D103, D104, D107,
+  D200, D203, D205, D209, D212, D213, D400
```
Comment (ksdme) [inline .coafile]: Is this the preferred way to break long list of items in a coafile?

Comment (gaocegege) [reply .coafile]: Have a look at https://github.com/coala/coala/blob/master/.coafile#L33

Comment (jayvdb) [reply .coafile]: ya, that is fine.

**Comment by jayvdb** [comment]
ack https://github.com/coala/coala-vs-code/pull/60/commits/6648e76bd6374ee826fdbf0a2aafa800d6c101bb https://github.com/coala/coala-vs-code/pull/60/commits/c48e6348092a5f0aad49dc9dc071c92534f8a8ea

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**